File: DownloadFiles/TGNet/Chapter03/VB/Exercise3-1/StoringTestDataPartI/FirstTestPartI/FirstTestPartI/Form1.vb
Imports System.IO
Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click

        Dim Writer As StreamWriter
        Writer = File.CreateText("TestResults.log")
        Writer.WriteLine("Test Results for Test Run XYZ; " & DateTime.Now)

        If File.Exists(TextBox1.Text) Then
            Writer.WriteLine("File Exists: Test Passed")
        Else
            Writer.WriteLine("File Exists: Test Failed")
        End If

        Writer.WriteLine("Test Completed for Test Run XYZ; " & DateTime.Now)
        Writer.Close()

    End Sub
End Class

File: DownloadFiles/TGNet/Chapter03/C#/Exercise3-1/StoringTestDataPartI/FirstTest/Form1.cs
#region Using directives

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Windows.Forms;
using System.IO;

#endregion

namespace FirstTest
{
    partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {

            StreamWriter Writer;
            Writer = File.CreateText("TestResults.log");
            Writer.WriteLine("Test Results for Test Run XYZ; " + DateTime.Now);
            if (File.Exists(textBox1.Text))
            {
                Writer.WriteLine("File Exists: Test Passed");
            }
            else
            {
                Writer.WriteLine("File Exists: Test Failed");
            }
            Writer.WriteLine("Test Completed for Test Run XYZ; " + DateTime.Now);
            Writer.Close();
        }
    }
}
